### PR TITLE
thomas 10/21

### PR DESCRIPTION
**Pull request by genlin**
Move billing content to the main repo

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@genlin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: articles/billing-buy-sign-up-azure-subscription.md.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<properties
+	pageTitle="How to buy or sign up for an Azure subscription"
```
Comment (tysonn) [inline articles/billing-buy-sign-up-azure-subscription.md.md]: Pls add the site identifier to the pageTitle value:
 | Microsoft Azure

File: articles/billing-buy-sign-up-azure-subscription.md.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	description="Describes How to buy or sign up for an Azure subscription"
+	services="billing"
+	documentationCenter=""
+	authors="genli"
+	manager="jarrettr"
+	editor="curtand"/>
```
Comment (tysonn) [inline articles/billing-buy-sign-up-azure-subscription.md.md]: Add another value to the properties section: 
tags="billing"

**Comment by tysonn** [comment]
genlin is a MSFT employee; I will be working on this pull request on Wednesday

**Comment by tysonn** [comment]
Moved to private repo.

**Pull request closed without merging**